packit/distgit.py

```diff
@@ -280,6 +280,9 @@ def push_to_fork(
                     "Unable to create a fork of repository "
                     f"{self.local_project.git_project.full_repo_name}",
                 )
+            # update fork ACL to allow users to push
+            for user in self.local_project.git_project.who_can_merge_pr():
+                fork.add_user(user, AccessLevel.push)
```

Comment:
nice! [The instructions](https://github.com/packit/packit/blob/main/packit/constants.py#L212-L217) should be updated to reflect this, right?

> It doesn't remove users who lost commit access to the original repo (unlikely to happen, but possible), do we want that?

it would be nice if it could always reflect that state of `who_can_merge_pr`, but as you say it is not very frequent situation, we can do it as followup

> It supports only users, not groups (there is no API in ogr for that)

we could add the missing support to ogr

Reply:
> nice! [The instructions](https://github.com/packit/packit/blob/main/packit/constants.py#L212-L217) should be updated to reflect this, right?

Yes, I'm working on that.